tests/models/gptsan_japanese/test_modeling_gptsan_japanese.py

```diff
@@ -152,7 +152,10 @@ def test_model(self):
         self.model_tester.create_and_check_model(*config_and_inputs)
 
     @unittest.skip(
-        reason="skip for now as the computed `max_memory` by `model_split_percents` in the test method will be changed inside `from_pretrained`"
+        reason=(
```

Comment:
I think all the changes inside `tests` shouldn't be there, can you please revert them !

Reply:
Done